op-service/sources/receipts_test.go

```diff
@@ -204,7 +204,7 @@ func randomRpcBlockAndReceipts(rng *rand.Rand, txCount uint64) (*rpcBlock, []*ty
 			Extra:       hexutil.Bytes(block.Extra()),
 			MixDigest:   block.MixDigest(),
 			Nonce:       types.EncodeNonce(block.Nonce()),
-			BaseFee:     (*hexutil.Big)(block.BaseFee()),
+			BaseFee:     new(hexutil.Big), //(*hexutil.Big)(block.BaseFee()),
```

Comment:
No need to skip its usage here either I think.

Reply:
done